pygpu/collectives.pyx

```diff
@@ -10,12 +10,21 @@ from pygpu.gpuarray import GpuArrayException
 
 
 cdef class GpuCommCliqueId:
+    """Represents a unique id shared among :ref:`GpuComm` communicators which
+    participate in a multi-gpu clique.
+
+    Parameters
+    ----------
+    context: :ref:`GpuContext`, optional
+        Reference to which gpu this `GpuCommCliqueId` object belongs.
+    comm_id: bytes, optional
+        Existing unique id to be passed in this object.
+
     """
-    """
-    def __cinit__(self, GpuContext context=None, bytes bytearr=None):
+    def __cinit__(self, GpuContext context=None, bytes comm_id=None):
         self.context = ensure_context(context)
-        if bytearr is not None:
-            self.set_comm_id(bytearr)
+        if comm_id is not None:
+            self.comm_id = comm_id
```

Comment:
I'm not sure since you have an attribute and a property for the same name.  In any case you shouldn't make python calls with self in `__cinit__` since the object is not valid yet.